src/test/java/apoc/es/ElasticSearchTest.java

```diff
@@ -11,42 +14,272 @@
 import java.net.ConnectException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.UUID;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
 
 /**
  * @author mh
  * @since 21.05.16
  */
 public class ElasticSearchTest {
 
-    private final static String HOST = "localhost";
     private final static String ES_INDEX = "test-index";
+
     private final static String ES_TYPE = "test-type";
-    private final static String ES_ID = "1";
 
-    private static GraphDatabaseService db;
+    private final static String ES_ID = UUID.randomUUID().toString();
 
+    private final static String HOST = "localhost";
+
+    protected static GraphDatabaseService db;
     // We need a reference to the class implementing the procedures
     private final ElasticSearch es = new ElasticSearch();
+    private Configuration JSON_PATH_CONFIG = Configuration.builder().options(Option.DEFAULT_PATH_LEAF_TO_NULL, Option.SUPPRESS_EXCEPTIONS).build();
 
     @BeforeClass
     public static void setUp() throws Exception {
         db = new TestGraphDatabaseFactory()
                 .newImpermanentDatabaseBuilder()
                 .newGraphDatabase();
         TestUtil.registerProcedure(db, ElasticSearch.class);
+        TestUtil.registerProcedure(db, Periodic.class);
+
+        insertDocuments();
     }
 
     @AfterClass
     public static void tearDown() {
         db.shutdown();
     }
 
+    private static void insertDocuments() {
+        TestUtil.ignoreException(() -> {
+            String payload = "{procedurePackage:'es',procedureName:'get',procedureDescription:'performa a GET operation to ElasticSearch'}";
+            String id = UUID.randomUUID().toString();
+
+            TestUtil.testCall(db, "CALL apoc.es.post('" + HOST + "','" + ES_INDEX + "','" + ES_TYPE + "','" + id + "',null," + payload + ") yield value", r -> assertNotNull(r.get("value")));
```

Comment:
you should use parameters here, not string concatenation, makes the queries much easier to read.